Sources/Swift/Integrations/SessionReplay/SentryReplayVideoMaker.swift

```diff
@@ -1,4 +1,4 @@
-#if canImport(UIKit)
```

Comment:
This reintroduced implicit UIKit linkage in https://github.com/getsentry/sentry-cocoa/pull/3877

Reply:
Don't we have tests to ensure we don't reintroduced implicit UIKit linkage?

Reply:
It wasn't checking for linkage with libswiftUIKit, which I added in https://github.com/getsentry/sentry-cocoa/pull/4140/commits/0cb91fbd63ca0483941f6c7febf10eefe1f7cd1d. 

This wasn't needed when I originally shipped the UIKitless stuff because you all weren't yet writing core SDK code in Swift. At the time you decided to do so, it should've been added.